backends/cuda-shared/ceed-cuda-shared.c

```diff
@@ -16,7 +16,6 @@
 
 #include <ceed/ceed.h>
 #include <ceed/backend.h>
-#include <stdbool.h>
 #include <string.h>
 #include "ceed-cuda-shared.h"
 #include "../cuda/ceed-cuda.h"
```

Comment:
One of these days we'll have to decide how we want this reaching over into other backend's headers to work.